src/cromshell/metadata/command.py

```diff
@@ -0,0 +1,124 @@
+import logging
+
+import click
+import requests
+
+from cromshell.utilities import cromshellconfig, http_utils, io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="metadata")
+@click.argument("workflow_id")
+@click.option(
+    "-des",
+    "--dont-expand-subworkflows",
+    is_flag=True,
+    default=True,
+    help="Do not expand subworkflow info in metadata",
+)
+@click.pass_obj
+def main(config, workflow_id: str, dont_expand_subworkflows: bool):
+    """Get the full metadata of a workflow."""
+
+    LOGGER.info("metadata")
+
+    check_cromwell_server(config=config, workflow_id=workflow_id)
+
+    obtain_and_print_metadata(
+        config=config,
+        metadata_param=config.METADATA_PARAMETERS,
+        exclude_keys=True,
+        expand_subworkflows=dont_expand_subworkflows,
```

Comment:
this will be confusing to readers of the code who doesn't know the history of the discussion of the PR.
let's rename the one in `obtain_and_print_metadata` to make it consistent

Reply:
👍